python/GafferSceneUI/PathFilterUI.py

```diff
@@ -114,6 +118,56 @@
 
 )
 
+##########################################################################
+# VectorDataPlugValueWidget customisation
+###########################################################################
+
+class _PathsPlugValueWidget( GafferUI.VectorDataPlugValueWidget ) :
+
+	def __init__( self, plug, **kw ) :
+
+		GafferUI.VectorDataPlugValueWidget.__init__( self, plug, **kw )
+
+		self.vectorDataWidget().dataMenuSignal().connect( Gaffer.WeakMethod( self.__dataMenu ), scoped = False )
+
+	def __dataMenu( self, vectorDataWidget, menuDefinition ) :
+
+		selectedIndices = vectorDataWidget.selectedIndices()
+
+		if len( selectedIndices ) > 0 :
+			menuDefinition.append( "/divider", { "divider" : True } )
+			menuDefinition.append(
+				"/Select Filtered Objects",
+				{
+					"command" : functools.partial( self.__selectAffected, selectedIndices )
+				}
+			)
+
+		return menuDefinition
+
+	def __selectAffected( self, selectedIndices ) :
+
+		filterNode = self.getPlug().node()
+
+		filterData = self.vectorDataWidget().getData()[0]
+		selectedStrings = [ filterData[row] for column, row in selectedIndices ]
+		selectedStrings = list( set( selectedStrings ) )
```

Comment:
I'd expect this to be redundant, because `pathMatcher.addPath()` will ignore duplicates.

Reply:
Fixed in 6bcd7db7c4b22ecc0d1e8c1a7ea1a1631a2ed230